client/devdocs/design/index.jsx

```diff
@@ -194,6 +197,10 @@ export default class DesignAssets extends Component {
 					<FAQ readmeFilePath="faq" />
 					<FeatureGate readmeFilePath="feature-example" />
 					<FilePickers readmeFilePath="file-picker" />
+					<FixedNavigationHeader
+						searchKeywords="header,navigatio,breadcrumbs"
```

Comment:
Actually, the `searchKeywords` prop seems to be unnecessary here as well.

Reply:
Basically, I would prefer if the FixedNavigationHeader comopnent shows up for `breadcrumbs` searches. Without searchKeywords it doesn't show up. What's your take? I have remove the rest of the keywords since they were included in the component name.

Reply:
Oh cool, that's totally fine if it's intentional 👍